### ISSUE TITLE
How to perform multimodal multitask instance segmentation in torchmultimodal?

### ISSUE THREAD

**Issue by Nandtiw** [facebookresearch/multimodal#444]
I have a dataset with 4 channel input imagery, 2 labels set - one label set is in MS COCO JSON Format and another labels format is in TIF format.
The dataset is designed for instance segmentation multi-modal multi-task.
Which model can be used for this task?
How do I give input of 2 labels to the instance segmentation model?
How to train for it?

Can someone please provide guidance on this task?

**Comment by pbontrager** [comment]
Thank you for your question. If you look under torchmultimodal/models you can see what models are included in multimodal. They are primarily content understanding with the exception of Dalle2. The only model included here that does segmentation is mdetr. You can look at examples/mdetr to see how to use it. MDETR expects text as it's label, so you'd want to format your two labels as if they were text. For the dataset, you can make a custom [Pytorch Dataset](https://pytorch.org/tutorials/beginner/basics/data_tutorial.html#creating-a-custom-dataset-for-your-files) and convert your labels into a format that works for MDETR inside of __getitem__.

**Comment by Nandtiw** [comment]
Thank you for your reply. I ll try this.

**Comment by Nandtiw** [comment]
Hi, I had a look into it. MDETR is for Object Detection + Text Feature Extractor (NLP).
Can MDETR be used for instance segmentation + instance segmentation tasks?
